packages/web-react/src/components/Footer/figma/Footer.figma.tsx

```diff
@@ -0,0 +1,280 @@
+import { Select } from '../../Select';
+import { Stack } from '../../Stack';
+import { VisuallyHidden } from '../../VisuallyHidden';
+import Footer from '../Footer';
+
+const FOOTER_NODE_URL = '<FIGMA_FILE_ID>?node-id=19410%3A3769';
+
+figma.connect(Footer, FOOTER_NODE_URL, {
+  variant: { 'Content below links': true },
+  props: {
+    logo: figma.boolean('Logo', {
+      true: (
+        <div className="text-desktop-left">
+          <Link href="https://www.example.com">
+            <ProductLogo>{defaultSvgLogo}</ProductLogo>
+          </Link>
+        </div>
+      ),
+      false: undefined,
+    }),
+    socialIcons: figma.boolean('Social icons', {
+      true: (
+        <Flex elementType="ul" alignmentX="center" alignmentY="center">
+          <li>
+            <ButtonLink size="medium" color="tertiary" isSymmetrical>
+              <VisuallyHidden>Facebook</VisuallyHidden>
+              <Icon name="logo-facebook" />
+            </ButtonLink>
+          </li>
+          <li>
+            <ButtonLink size="medium" color="tertiary" isSymmetrical>
+              <VisuallyHidden>X</VisuallyHidden>
+              <Icon name="logo-x" />
+            </ButtonLink>
+          </li>
+          <li>
+            <ButtonLink size="medium" color="tertiary" isSymmetrical>
+              <VisuallyHidden>YouTube</VisuallyHidden>
+              <Icon name="logo-youtube" />
+            </ButtonLink>
+          </li>
+        </Flex>
+      ),
+      false: undefined,
+    }),
+    language: figma.boolean('Language', {
+      true: (
+        <div className="text-desktop-right">
+          <Select id="select-language" name="selectLanguage" label="Language" isLabelHidden>
+            <option value="en">English</option>
+            <option value="cs">Čeština</option>
+          </Select>
+        </div>
+      ),
+      false: undefined,
+    }),
+    secondaryLinks: figma.boolean('Secondary links', {
+      true: (
+        <>
+          <Divider marginY="space-1200" />
+          <nav aria-label="Secondary links">
+            <Flex
+              elementType="ul"
+              direction={{ mobile: 'column', tablet: 'row' }}
+              alignmentX={{ mobile: 'stretch', tablet: 'center' }}
+              spacing={{ mobile: 'space-600', tablet: 'space-900' }}
+              isWrapping
+            >
+              <li>
+                <Link href="https://www.example.com" color="secondary">
+                  Legal notice
+                </Link>
+              </li>
+              <li>
+                <Link href="https://www.example.com" color="secondary">
+                  Terms of service
+                </Link>
+              </li>
+              <li>
+                <Link href="https://www.example.com" color="secondary">
+                  Privacy policy
+                </Link>
+              </li>
+            </Flex>
+          </nav>
+        </>
+      ),
+      false: undefined,
+    }),
+    showRow2: figma.boolean('Show Row 2', {
```

Comment:
And where is row1? :)

Could you please fix the order of the footer subcomponents? 🙏🏻

Reply:
figma naming 🤷

Reply:
order fixed